packages/components/src/components/Snackbar/Snackbar.tsx

```diff
@@ -39,7 +41,7 @@ type snackbarData = {
  *
  * In the future, this may change (e.g. allowing non-indefinite `duration`)
  */
-type modifyToastOptions = snackbarData
+export type modifyToastOptions = snackbarData
```

Comment:
Instead of exporting, I think this can just be deleted. And `SnackbarType` below deleted as it's no longer used without the global.